helm-chart/renku/Chart.yaml

```
@@ -3,4 +3,4 @@ appVersion: '1.0'
 description: A Helm chart for the Renku platform
 name: renku
 icon: https://github.com/SwissDataScienceCenter/renku-sphinx-theme/raw/master/renku_sphinx_theme/static/favicon.png
-version: 0.17.2
+version: 0.18.0
```

```suggestion
version: 0.0.1-set.by.chartpress
```

This has been updated manually for a while now. But we should leave it to a fixed value like this and not touch it. When we release the helm chart chartpress will assign the right tag.